test/spec/adapters/rubicon_spec.js

```diff
@@ -0,0 +1,292 @@
+
+      expect(bidderRequest).to.have.deep.property("bids[0]")
+        .to.have.property("bidder", "rubicon");
+
+      expect(bidderRequest).to.have.deep.property("bids[0]")
+        .to.have.property("placementCode", adUnit.code);
+
+      expect(bidderRequest).to.have.deep.property("bids[0]")
+        .with.property("sizes")
+        .that.is.an("array")
+        .with.lengthOf(2)
+        .that.deep.equals(adUnit.sizes);
+
+      expect(bidderRequest).to.have.deep.property("bids[0]")
+        .with.property('params')
+        .that.deep.equals(adUnit.bids[0].params)
+
+    });
+
+  });
+
+  describe("callBids implementation", () => {
+
+    let bidderRequest,
+        slot;
+
+    beforeEach(() => {
+      sandbox.stub(adloader, "loadScript");
+      sandbox.spy(rubiconAdapter, "callBids");
+
+      slot = {
+        clearTargeting: sandbox.spy(),
+        setPosition: sandbox.spy(),
+        addFPV: sandbox.spy(),
+        addFPI: sandbox.spy(),
+        addKW: sandbox.spy(),
+        getElementId: () => "/19968336/header-bid-tag-0",
+        getRawResponses: () => {}
+      };
+
+      window.rubicontag = {
+        cmd: {
+          push: cb => cb()
+        },
+        setIntegration: sandbox.spy(),
+        run: () => {},
+        setUserKey: sandbox.spy(),
+        defineSlot: sandbox.spy(bid => slot)
+      };
+
+      bidderRequest = {
+        bidderCode: "rubicon",
+        requestId: "c45dd708-a418-42ec-b8a7-b70a6c6fab0a",
+        bidderRequestId: "178e34bad3658f",
+        bids: [
+          {
+            bidder: "rubicon",
+            params: {
+              accountId: "14062",
+              siteId: "70608",
+              zoneId: "335918",
+              userId: "12346",
+              keywords: ["a","b","c"],
+              inventory: {
+                rating:"5-star",
+                prodtype:"tech"
+              },
+              visitor: {
+                ucat:"new",
+                lastsearch:"iphone"
+              },
+              position: "atf"
+            },
+            placementCode: "/19968336/header-bid-tag-0",
+            sizes: [[300, 250], [320, 50]],
+            bidId: "2ffb201a808da7",
+            bidderRequestId: "178e34bad3658f",
+            requestId: "c45dd708-a418-42ec-b8a7-b70a6c6fab0a"
+          }
+        ],
+        start: 1472239426002,
+        timeout: 5000
+      };
+
+    });
+
+    describe("when doing fastlane slot configuration", () => {
+
+      beforeEach(() => {
+        rubiconAdapter.callBids(bidderRequest);
+      });
+
+      it("should load the fastlane SDK if not loaded", () => {
+
+        let pathToSDK = adloader.loadScript.getCall(0).args[0];
+        expect(pathToSDK).to.equal(`http://ads.rubiconproject.com/header/${bidderRequest.bids[0].params.accountId}.js`);
+
+        rubiconAdapter.callBids(bidderRequest);
+        expect(adloader.loadScript.calledOnce).to.equal(true);
+
+      });
+
+      it("should make a valid call to rubicontag.defineSlot", () => {
```

Comment:
can probably remove 'should' from each of the it()'s

should make a valid call ->  makes a valid call to (or even just.. calls to)
should call rubicontag.setUserKey -> calls rubicontag.setUerKey
should set proper targeting params  ->  sets targeting params

Not really an issue requiring a change, just a thought